core/backend/src/rpc-impl/IModelReadRpcImpl.ts

```diff
@@ -31,20 +31,24 @@ export class IModelReadRpcImpl extends RpcInterface implements IModelReadRpcInte
   public static register() { RpcManager.registerImpl(IModelReadRpcInterface, IModelReadRpcImpl); }
 
   public async getConnectionProps(tokenProps: IModelRpcOpenProps): Promise<IModelConnectionProps> {
-    return RpcBriefcaseUtility.openWithTimeout(RpcTrace.currentActivity!, tokenProps, SyncMode.FixedVersion);
+    assert(undefined !== RpcTrace.currentActivity);
```

Comment:
instead of repeating this assert everywhere, lets make  `RpcTrace.expectCurrentActivity`

Reply:
I tried to create this assertion function but found that TypeScript assertions can only assert arguments passed into the function. What about an alternative getter that asserts then returns a non-null RpcActivity?

Reply:
@pmconne, @kabentley, Got it - change is committed.